packages/app-mobile/utils/fs-driver-rn.ts

```diff
@@ -131,6 +209,10 @@ export default class FsDriverRN extends FsDriverBase {
 
 	public async unlink(path: string) {
 		try {
+			if (isScopedUri(path)) {
+				await RNSAF.unlink(path);
```

Comment:
Did you handle the error below? i.e. if the file doesn't exist, does your lib throw an exception? And if so, it needs to be handled below - in this case it should be a noop

Reply:
done